src/ui/Base/RegisteredNamedMethods.ts

```diff
@@ -447,6 +453,54 @@ Initialization.registerNamedMethod('configureResourceRoot', (path: string) => {
   configureResourceRoot(path);
 });
 
+/**
+ * Re-enables all [Analytics]{@link Analytics} components if they were previously disabled.
+ * @param searchRoot What ancestor or CoveoAnalytics element to search from in the DOM.
+ */
+export function enableAnalytics(searchRoot = document.querySelector(Component.computeSelectorForType(SearchInterface.ID)) as HTMLElement) {
+  const analytics = getCoveoAnalytics(searchRoot) as Analytics;
+  if (!analytics) {
+    return registeredNamedMethodsLogger.warn('Could not enable analytics because they could not be found.');
+  }
+  analytics.enable();
+}
+
+Initialization.registerNamedMethod('enableAnalytics', () => {
+  enableAnalytics();
+});
+
+/**
+ * Removes all session information stored in the browser (e.g. analytics visitor cookies, action history, etc.)
```

Comment:
See comment in [Analytics.ts](https://github.com/coveo/search-ui/pull/1214/files#diff-d2533c6cdeeedf558edab08f57cf9546R387)